src/utils.ts

```diff
@@ -560,6 +560,11 @@ export function deprecateOptions(
   return deprecated;
 }
 
+/** @internal */
+export function ns(ns: string): MongoDBNamespace {
+  return MongoDBNamespace.fromString(ns);
+}
```

Comment:
👏 👏 👏  "Use namespace through and through" TODOs beware!
Did this fix all of those?

Reply:
Hm, I've been trying to make that change as much as possible, but I don't think we can assume everything is typed as such. Someone should manually check that a string is never being used